### PR TITLE
Northstar tl to quasar

### PR DESCRIPTION
**Pull request by sheyd**
#### What's this PR do?
Provides ETL script for Northstar -> Quasar ingestion.

#### Where should the reviewer start?
Only one file present! At the beginning. 

#### How should this be manually tested?
Already tested in Prod.

#### Any background context you want to provide?
Current version sets page max from Northstar on initial request. @DFurnes had the recommendation for the script to use the last request page per request to NS as a basis. I'd like to build this into a future version, but currently prefer the bounded static page approach, as the script has some issues with long-running jobs. 

#### What are the relevant tickets?
#294

### CODE REVIEW

File: etl-scripts/northstar_to_user_table.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+if len(sys.argv) < 2:
+    cur.execute("SELECT * from quasar_etl_status.northstar_ingestion WHERE counter_name = 'last_page_scraped';")
+    db.commit()
+    last_page = cur.fetchall()
+    i = last_page[0][1]
+else:
+    i = int(sys.argv[1])
+"""Check if page starting point provided, otherwise use last processed point."""
+
+def ts(base_value):
```
Comment (mshmsh5000) [inline etl-scripts/northstar_to_user_table.py]: Low priority: Recommend using method/function names that are more self-evident, e.g., `to_string`

File: etl-scripts/northstar_to_user_table.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    """Converts to string and replaces values with NULL when blank or None."""
+    base_string = str(base_value)
+    strip_special_chars = re.sub(r'[()<>/"\'\\]','',base_string)
+    transform_null = re.sub(r'^$|\bNone\b','NULL',strip_special_chars)
+    transformed = str(transform_null)
+    return transformed
```
Comment (mshmsh5000) [inline etl-scripts/northstar_to_user_table.py]: Again, low priority: Could condense these two lines into one, viz.,

```
return str(transform_null)
```

**Comment by mshmsh5000** [comment]
Generally 👍 ❗️ 

I recommend trying a linter pre-commit hook, e.g., https://github.com/cbrueffer/pep8-git-hook — should catch a bunch of formatting & minor stuff for you.

File: etl-scripts/northstar_to_user_table.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+import sys
+from DSNorthstarScraper import NorthstarScraper
+
+"""DS Northstar to Quasar User ETL script.
+
+This ETL scripts scrapes the DoSomething Northstar User API and ETL's the
+output to our MySQL Quasar data warehouse.
+
+The script takes an optional argument for what Northstar page result to start
+on. This is mostly used to backfill from a certain page, or from the dawn
```
Comment (DFurnes) [inline etl-scripts/northstar_to_user_table.py]: the dawn of time

File: etl-scripts/northstar_to_user_table.py [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+db = MySQLdb.connect(host=config.host,  #  hostname
+          user=config.user,             #  username
+          passwd=config.pw)             #  password
+
+db.set_character_set('utf8')
+cur = db.cursor()
+cur.execute('SET NAMES utf8;')
+cur.execute('SET CHARACTER SET utf8;')
+cur.execute('SET character_set_connection=utf8;')
+"""Set UTF-8 encoding on MySQL connection."""
```
Comment (DFurnes) [inline etl-scripts/northstar_to_user_table.py]: I think these should be formatted as [block comments](https://www.python.org/dev/peps/pep-0008/#block-comments) above the relevant block of code, not [docstrings](https://www.python.org/dev/peps/pep-0008/#documentation-strings). 👾

**Comment by sheyd** [comment]
Woohoo, no lint errors! 👕  Merging.

**Pull request merged**